itou/templates/layout/_footer.html

```diff
@@ -104,6 +104,9 @@
                         <li>
                             <a href="{% url 'accessibility' %}">Accessibilité : partiellement conforme</a>
                         </li>
+                        <li>
+                            <a href="{% static 'pdf/syntheseSecurite.pdf' %}" rel="noopener" target="_blank">Sécurité : Homologation avec MonService Sécurisé</a>
```

Comment:
https://monservicesecurise.cyber.gouv.fr/
```suggestion
                            <a href="{% static 'pdf/syntheseSecurite.pdf' %}" rel="noopener" target="_blank">Sécurité : Homologation avec MonServiceSécurisé</a>
```